scripts/hid_configurator/NrfHidManager.py

```diff
@@ -9,7 +9,7 @@
 
 class NrfHidManager():
     TYPE2BOARDLIST = {
-        'gaming_mouse' : ['nrf52840gmouse', 'nrf52840dk', 'nrf54l15pdk', 'nrf54h20dk'],
+        'gaming_mouse' : ['nrf52840gmouse', 'nrf52840dk', 'nrf54l15dk', 'nrf54h20dk'],
```

Comment:
If we are not removing pdk from code I wouldn't remove it from this script. Lets for now keep both here.

Reply:
Changed. Then the PDK will be removed only from documentation.